### PR TITLE
service friends added to the reuse card

### PR DESCRIPTION
**Pull request by matthewhall78**
# Description
3 images, for getok (tokey), cdogs (sam), and ches (missy) added the reuse card. added hyperlinks to the images pointing to each of the card's page anchor.

## Types of changes

Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: _cards/features.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ urls:
   getstarted: "#Onboarding"  
 ---
 <div class="row">
-  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-xs-12">Using common services will jump start the development efforts of your team and lower the overall maintenance cost of your product. You can focus on building features that matter to your users. Get started today!</div>
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-xs-12">
```
Comment (jujaga) [inline _cards/features.md]: I can't quite remember bootstrap, but I thik you might be able to simplify this to just
```
<div class="col-lg-6 col-xs-12">
```
As the default is to look at the smallest box size and apply that upwards until a different rule applies.

Comment (loneil) [inline _cards/features.md]: Can be replaced with "col-lg-6 col-xs-12" as only the lower-bound breakpoint is required

File: _cards/reuse.md [hunk 2]
```diff
@@ -4,12 +4,35 @@ title: Why do the work when we've done it for you?
...
+    alt: "Doc Generation" 
+    url: "#CDOGS"
 ---
 <div class="row">
-  <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12">There is a set common services ready for you to use today. These services are reliable, highly available, and supported by an experienced team of developers on an Agile team in IITD. Check out the APIs and showcase applications below to continue with the onboarding process</div>
-  <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12">
-          {% if page.img %}
-            <img class="img-fluid my-3" src="{{ site.baseurl }}{{page.img.path }}" alt="{{ page.img.alt }}">
-          {% endif %}   
+  <div class="col-xl-5 col-lg-5 col-md-12 col-sm-12 col-xs-12">
```
Comment (loneil) [inline _cards/reuse.md]: "col-lg-5 col-xs-12"

File: _cards/reuse.md [hunk 3]
```diff
@@ -4,12 +4,35 @@ title: Why do the work when we've done it for you?
...
+    <p>There is a set common services ready for you to use today. </p>
+    <p>These services are reliable, highly available, and supported by an experienced team of developers on an Agile team in IITD. </p>
+    <p><strong>Check out the APIs and showcase applications below to continue with the onboarding process</strong></p>
+  </div>
+  <div class="col-xl-7 col-lg-7 col-md-12 col-sm-12">
```
Comment (loneil) [inline _cards/reuse.md]: "col-lg-7 col-sm-12"

**Review by loneil — CHANGES_REQUESTED** [review]
Just a note on bootstrap grid setting duplication

File: _cards/features.md (hunk 1 above)
Comment (matthewhall78) [reply _cards/features.md]: done

File: _cards/reuse.md (hunk 2 above)
Comment (matthewhall78) [reply _cards/reuse.md]: done

**Review by jujaga — APPROVED** [review]

**Review by loneil — APPROVED** [review]
That'll do it

**Pull request merged**